tests/js/spec/views/performance/vitalDetail/index.spec.jsx

```diff
@@ -1,346 +0,0 @@
-        location={initialData.router.location}
-      />,
-      initialData.routerContext
-    );
-    await tick();
-    wrapper.update();
-
-    // Fill out the search box, and submit it.
-    const searchBar = wrapper.find('SearchBar textarea');
-    searchBar
-      .simulate('change', {target: {value: 'user.email:uhoh*'}})
-      .simulate('submit', {preventDefault() {}});
-
-    // Check the navigation.
-    expect(browserHistory.push).toHaveBeenCalledTimes(1);
-    expect(browserHistory.push).toHaveBeenCalledWith({
-      pathname: undefined,
-      query: {
-        project: 1,
-        statsPeriod: '14d',
-        query: 'user.email:uhoh*',
-      },
-    });
-  });
-
-  it('Applies conditions when linking to transaction summary', async function () {
-    const initialData = initializeData({
-      query: {
-        query: 'sometag:value',
-      },
-    });
-    const wrapper = mountWithTheme(
-      <WrappedComponent
-        organization={initialData.organization}
-        location={initialData.router.location}
-      />,
-      initialData.routerContext
-    );
-    await tick();
-    wrapper.update();
-
-    const firstTransactionFromList = wrapper.find('Table GridBody GridRow Link').at(1);
-
-    expect(firstTransactionFromList.prop('to')).toEqual(
-      expect.objectContaining({
-        pathname: '/organizations/org-slug/performance/summary/',
-        query: expect.objectContaining({
-          display: 'vitals',
-          query: 'sometag:value has:measurements.lcp',
-          showTransactions: 'recent',
-          statsPeriod: '24h',
-          transaction: 'something',
-        }),
-      })
-    );
-  });
-
-  it('Check CLS', async function () {
-    const initialData = initializeData({
-      query: {
-        query: 'anothertag:value',
-        vitalName: 'measurements.cls',
-      },
-    });
-    const wrapper = mountWithTheme(
-      <WrappedComponent
-        organization={initialData.organization}
-        location={initialData.router.location}
-      />,
-      initialData.routerContext
-    );
-    await tick();
-    wrapper.update();
-
-    expect(wrapper.find('Title').text()).toEqual('Cumulative Layout Shift');
-
-    const firstTransactionFromList = wrapper.find('Table GridBody GridRow Link').at(1);
-
-    expect(firstTransactionFromList.prop('to')).toEqual(
-      expect.objectContaining({
-        pathname: '/organizations/org-slug/performance/summary/',
-        query: expect.objectContaining({
-          display: 'vitals',
-          query: 'anothertag:value has:measurements.cls',
-          showTransactions: 'recent',
-          statsPeriod: '24h',
-          transaction: 'something',
-        }),
-      })
-    );
-
-    // Check cells are not in ms
-    const firstRow = wrapper.find('GridBody GridRow').first();
-    expect(firstRow.find('GridBodyCell').at(6).text()).toEqual('0.215');
-  });
-
-  it('Pagination links exist to switch between vitals', async function () {
```

Comment:
Is there a reason this test was not migrated?

Reply:
no... the commit was not pushed on Friday...it shall be fine now